src/jennifer/model/mapping.cr

```diff
@@ -320,23 +320,29 @@ module Jennifer
           end
         {% end %}
 
-        def save(skip_validation = false)
+        def save!(skip_validation = false)
+          res = save(skip_validation)
+          raise Jennifer::BaseException.new("Record was not save. Error list: #{errors.inspect}") unless res
```

Comment:
typo, you probably mean
`Record was not saved`

Reply:
Thanks